app/models/job.rb

```diff
@@ -20,4 +20,10 @@ class Job < ActiveRecord::Base
             presence: true,
             inclusion: { in: STATES }
   validates :author, presence: true
+
+  def self.search(search)
```

Comment:
In most recent versions of Rails the preference for using a named scope is subject to personal preference. This is extracted from the [guides](http://guides.rubyonrails.org/active_record_querying.html#scopes):

>This is exactly the same as defining a class method, and which you use is a matter of personal preference.

As long as a model method returns an `ActiveRecord::Relation` it can be chained, etc. Scopes might be more readable but, methods are easier to extract. It's Rails versus Ruby.

This is the implementation for named scopes in `activerecord`. https://github.com/rails/rails/blob/010e246756c09f44e901f4fd8e8eab2cb3022e95/activerecord/lib/active_record/scoping/named.rb#L143